rpi_backlight.py

```diff
@@ -16,7 +16,8 @@
 
 __author__ = "Linus Groh"
 __version__ = "1.8.1"
-PATH = "/sys/class/backlight/rpi_backlight/"
+path = ""
```

Comment:
Please default to the original path so calling `init` is optional in order to remain backwards compatible. Also `path` -> `PATH`.

Reply:
I was not able to get it to work with constants, only global variables. Maybe you could give it a shot. I spent an hour with constants before giving up

Reply:
There's no such thing as real constants in python - it's just a naming convention. Having an uppercase variable won't magically remove the ability to change its value.

Reply:
Fixed the defaults

Reply:
Fixed